README.md

```diff
@@ -43,6 +43,15 @@ npm run dev
 ```
 
 - API is started on http://localhost:3060
+Request can now be posted to http://localhost:3060/graphql/v2 like 
+
+`curl --request POST \
+  --header 'content-type: application/json' \
+  --url 'http://localhost:3060/graphql/v2' \
+  --data '{"query":"query { __typename }"}'``
+
+```
+
 - A local email inbox is started on http://localhost:1080
 
```

Comment:
I don't find this useful, `curl` can be nice to ping the API but it's definitely not practical to query the API. We have some [docs](https://github.com/opencollective/opencollective-api/blob/d5646f6768aef1910e1fc472e6f5f1f096462cec/docs/dev.md) about querying basics, I would rather link there and keep this one simple:



```suggestion
- A local email inbox is started on http://localhost:1080

See the [dev docs](docs/dev.md) for querying basics.
```